### ISSUE TITLE
Selenium tests for basic UI

### ISSUE THREAD

**Issue by krzysztof** [zenodo/zenodo#480]
Add selenium tests for some of the UI modules, such as: login, registration, user profile.

**Comment by rahulyadav170923** [comment]
@krzysztof i would like to work on it. Is there an order for preparing the tests you would prefer. Or should i start writing for the UI elements one by one.

**Comment by krzysztof** [comment]
Currently our selenium tests are very limited: https://github.com/zenodo/zenodo/tree/master/tests/e2e :wink: 

We don't have a preference but I guess it's good to start with some easy to test pages like the login page, registration, perhaps a multi-step registration form click-through etc. Then we'd want some of the "crucial" pages like the search results, record page with the previewed file etc.

**Comment by krzysztof** [comment]
The final boss would be to test the deposit publishing from A to Z: creating a new deposit, publishing, editing it with a small change, then publishing again.

**Comment by rahulyadav170923** [comment]
@krzysztof @slint i have started working on it but cannot contribute before 10 March. due to my exams so if anyone want to work on it . It is free as of now.

**Comment by krzysztof** [comment]
@rahulyadav170923 No worries, thanks a lot for freeing it for others! :+1:

**Comment by kishansag** [comment]
@krzysztof Since Rahul is busy I would like to start working on this one.

**Comment by krzysztof** [comment]
@kishansag :+1:

**Comment by krzysztof** [comment]
Some examples of E2E tests from Invenio-Accounts module:

https://github.com/inveniosoftware/invenio-accounts/blob/master/tests/e2e/e2e_basic_test.py

login and registration is already there, we should "borrow" it as to make sure it still works with Zenodo's overlay/theme

**Comment by amarlearning** [comment]
@krzysztof Anyone working on this ?

**Comment by rajat1994** [comment]
@amarlearning ... yup

**Comment by kishansag** [comment]
yup

**Comment by krzysztof** [comment]
@amarlearning Yes, but there are many UI tests to be done though, so if you coordinate with other students (I'd suggest), you can select some UI pages which are not tested. Some of the static pages are being tested, but AFAIK search is not. Take a look at currently work in progress "selenium" PRs:
https://github.com/zenodo/zenodo/pulls

**Comment by DeadlyHollows** [comment]
Can I also contribute with you guys if there is room?

**Comment by kishansag** [comment]
@DeadlyHollows Sure man. I was getting errors in travis checks, so have stopped working on this. You can start with login or registrations. 
If you want to use my code you can find it here https://github.com/kishansag/zenodo/tree/testing/tests/e2e
or you can also borrow code from https://github.com/inveniosoftware/invenio-accounts/blob/master/tests/e2e/e2e_basic_test.py

**Comment by rajat1994** [comment]
@DeadlyHollows ... are you working on this?
